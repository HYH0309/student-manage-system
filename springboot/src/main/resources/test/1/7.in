201
7 1 1
10 17 6 20 10 11 15 
2 7 7
17 18 
0 0 5

0 18 5

0 6 5

3 10 0
13 2 10 
3 15 2
11 4 14 
0 17 0

1 17 6
6 
3 6 5
14 13 19 
5 5 2
20 3 5 9 12 
2 14 4
15 12 
0 1 1

6 19 1
17 17 12 13 12 3 
6 7 5
13 19 17 20 4 19 
7 13 5
2 10 19 12 7 17 19 
7 13 6
1 11 1 1 20 16 5 
4 15 4
1 13 9 1 
2 11 1
13 8 
1 7 0
2 
4 13 5
17 9 20 2 
6 1 6
8 3 20 2 10 11 
2 8 7
9 10 
2 19 0
11 13 
0 16 5

0 6 7

6 19 6
15 3 11 20 2 17 
0 3 2

4 20 7
7 5 3 15 
0 18 7

3 3 4
19 19 13 
2 6 3
3 13 
4 0 3
16 2 15 4 
1 14 3
16 
0 15 0

4 13 4
3 6 17 4 
3 1 5
18 5 3 
6 20 0
5 6 8 14 8 4 
7 5 1
4 11 9 14 2 3 6 
3 9 6
20 13 1 
7 12 6
15 8 9 12 2 7 2 
4 15 0
9 19 8 18 
2 11 5
1 12 
1 0 6
5 
4 2 0
9 13 13 2 
1 8 3
10 
7 15 1
2 14 8 13 9 8 6 
0 9 6

6 12 6
6 17 1 20 15 10 
4 6 2
15 3 14 12 
0 20 2

3 19 5
12 10 9 
6 0 6
19 15 16 1 14 13 
3 17 1
17 9 10 
5 16 4
18 12 8 18 14 
2 4 5
3 7 
7 17 7
12 11 12 19 16 14 6 
6 9 4
6 7 14 4 8 5 
7 14 5
8 15 20 16 6 20 3 
1 18 3
14 
3 10 4
10 16 20 
0 17 5

4 1 2
17 2 1 12 
4 3 3
7 10 11 18 
2 15 7
3 19 
2 4 1
6 3 
2 20 7
2 14 
7 5 3
6 5 7 14 5 1 19 
5 18 2
15 17 15 14 14 
4 18 4
4 10 2 12 
3 1 6
3 18 12 
0 17 1

2 10 1
4 15 
6 16 6
5 14 9 2 1 7 
6 10 4
10 3 10 13 17 16 
1 7 4
14 
0 15 7

5 15 2
19 6 18 12 16 
3 14 5
16 18 20 
5 13 6
15 14 14 16 9 
7 3 4
20 16 9 10 17 3 17 
5 11 6
1 17 1 3 12 
3 5 4
13 4 20 
0 10 7

5 8 6
19 8 16 17 20 
3 16 5
3 11 15 
2 5 5
12 14 
6 13 2
3 11 16 6 16 17 
3 8 6
11 18 9 
2 0 6
1 2 
5 7 0
20 10 2 7 20 
2 1 1
2 11 
0 17 2

7 8 6
17 15 3 8 10 8 20 
4 15 7
2 4 10 16 
6 4 1
5 20 11 3 20 5 
4 19 2
20 19 8 6 
5 15 5
20 2 13 4 16 
4 8 5
9 1 9 3 
2 20 5
11 16 
3 13 7
17 6 17 
0 17 6

6 7 5
20 17 6 13 15 13 
6 1 6
14 9 14 6 10 9 
0 4 0

4 16 3
1 20 1 4 
6 9 1
8 10 2 1 6 8 
0 12 5

2 3 7
2 19 
0 16 1

5 0 4
2 1 5 7 4 
4 7 2
1 8 17 19 
1 2 0
17 
5 0 0
12 19 7 9 12 
2 6 3
10 1 
2 4 4
8 1 
3 14 3
12 4 14 
2 4 3
15 15 
1 16 7
12 
1 10 6
11 
6 12 6
10 20 2 16 12 9 
0 5 4

6 18 3
4 5 14 4 13 11 
2 13 6
7 19 
4 12 5
7 14 2 12 
4 9 0
9 1 4 12 
2 10 2
8 12 
2 19 7
10 3 
1 12 3
8 
4 13 1
20 4 18 18 
6 4 7
13 20 6 3 7 9 
6 2 3
7 2 8 8 20 17 
1 9 4
20 
7 15 5
2 19 17 8 11 18 18 
4 15 3
9 15 5 2 
5 19 1
12 17 6 8 6 
3 3 0
9 2 18 
2 12 0
17 2 
2 14 6
2 20 
7 1 4
2 9 17 16 11 11 9 
0 13 6

3 2 4
1 9 3 
4 8 4
18 15 12 15 
4 14 3
7 4 20 11 
0 5 3

2 12 2
6 13 
6 14 2
18 5 18 17 11 4 
5 7 4
10 1 14 1 17 
5 17 6
14 9 10 4 13 
0 18 6

3 18 7
17 17 20 
0 15 1

1 5 7
2 
0 11 5

1 3 3
19 
3 11 7
8 18 6 
7 9 1
4 11 2 9 11 3 3 
3 10 5
15 1 18 
5 3 0
13 20 12 3 20 
3 2 3
19 11 10 
3 3 0
19 1 9 
2 14 1
16 11 
1 5 7
1 
3 0 2
11 6 7 
1 19 6
13 
4 6 7
11 14 14 15 
1 16 3
16 
5 3 5
15 6 12 3 5 
0 12 6

4 16 2
18 5 2 5 
1 5 5
16 
7 1 1
17 7 8 6 18 13 6 
0 17 0

3 13 1
18 17 12 
3 3 4
16 7 1 
1 10 5
11 
5 1 6
14 20 1 3 10 
4 1 5
7 13 14 18 
5 6 6
10 3 9 3 6 
5 18 2
15 15 16 7 4 
6 14 3
9 3 20 15 9 13 
4 7 4
15 10 16 5 
4 17 3
16 1 6 1 
0 1 3

2 6 3
4 6 
1 8 6
14 
7 16 6
17 20 14 1 10 8 15 
6 3 7
18 2 2 13 11 13 
1 12 4
4 
5 4 0
1 8 14 20 11 
2 5 5
1 5 
3 11 3
7 5 4 
5 0 0
13 2 2 12 2 
0 12 1

6 17 6
15 3 5 12 14 16 
7 18 3
2 15 6 10 10 1 3 
1 11 0
20 
5 9 1
20 19 16 4 11 
7 10 6
17 20 9 8 3 9 13 
7 5 3
5 8 8 13 11 13 7 
2 10 0
15 8 
