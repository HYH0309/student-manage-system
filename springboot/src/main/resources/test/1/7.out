djwcb
-1
djwcb
-1
djwnb
0
djwnb
0
djwnb
0
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwnb
0
djwcb
-1
djwnb
0
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwnb
0
djwnb
2
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwnb
0
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
djwcb
-1
